spec/rubocop/cop/internal_affairs/empty_line_between_expect_offense_and_correction_spec.rb

```diff
@@ -90,4 +90,10 @@
       expect_no_corrections
     RUBY
   end
+
+  it 'does not register an offense for `expect_offense` with unexpected arguments' do
```

Comment:
It seems that terms like `unexpected syntax` or `invalid usage` would be more appropriate:

```suggestion
  it 'does not register an offense for `expect_offense` with unexpected syntax' do
```